packages/bpk-component-select/stories.js

```diff
@@ -17,11 +17,96 @@
  */
 
 import React from 'react';
+import PropTypes from 'prop-types';
 import { storiesOf } from '@storybook/react';
 import { action } from '@storybook/addon-actions';
 
 import BpkSelect from './index';
 
+class SelectWithImage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selected: 0,
+    };
+  }
+
+  getItemByValue = () => {
+    const { options } = this.props;
+    return val => options.filter(o => o.value === val);
+  };
+
+  getItem = this.getItemByValue();
+
+  handleChange = e => {
+    const [item] = this.getItem(e.target.value);
+
+    if (!item) return;
+
+    this.setState({
+      selected: item.key,
+    });
+  };
+
+  image = src => () => <img alt="fruit" style={{ width: '100%' }} src={src} />;
+
+  render() {
+    const { options, ...rest } = this.props;
+    return (
+      <BpkSelect
+        value={options[this.state.selected].value}
+        {...rest}
+        image={this.image(options[this.state.selected].image)}
+        onChange={this.handleChange}
+      >
+        {options.map(o => (
+          <option
+            key={o.value}
+            disabled={o.disabled && 'disabled'}
+            value={o.value}
+          >
+            {o.label}
+          </option>
+        ))}
+      </BpkSelect>
+    );
+  }
+}
+
+SelectWithImage.propTypes = {
+  options: PropTypes.arrayOf(PropTypes.object).isRequired,
+};
+
+const options = [
+  {
+    key: 0,
+    value: 'apple',
+    label: 'Apple',
+    image: 'http://via.placeholder.com/250x50/ff5452/000000',
+    disabled: false,
+  },
+  {
+    key: 1,
+    value: 'oranges',
+    label: 'Oranges',
+    image: 'http://via.placeholder.com/250x50/ffbb00/000000',
+    disabled: false,
+  },
+  {
+    key: 2,
+    value: 'pear',
+    label: 'Pear',
+    image: 'http://via.placeholder.com/250x50/00d775/000000',
+    disabled: false,
+  },
+  {
+    key: 3,
+    value: 'tomato',
+    label: 'Tomato',
+    image: 'http://via.placeholder.com/250x50/ff5432/000000',
```

Comment:
I'm wondering if we should refrain from using another external service here. 

In the `FlatList` component [we use our own CDN to grab images for use in the storybook](https://github.com/Skyscanner/backpack/blob/master/native/packages/react-native-bpk-component-flat-list/stories.js#L39), maybe that'd be good here too (we could change the fruits to countries). It's still an 'external' service, but then we'd only be reliant on one service.

What do you think?